RAGchain/preprocess/loader/hwp_table_loader.py

```diff
@@ -0,0 +1,152 @@
+import os
+import re
+import zipfile
+import shutil
+from typing import List
+
+import win32com.client as win32
+from bs4 import BeautifulSoup
```

Comment:
Plus, beautifulsoup is not requirements of RAGchain.
But do not add this package to setup.py. Just make try-except, and if bs4 is not installed, raise Error that beautifulsoup not installed.